### DISCUSSION TITLE
baseline correct tau

### DISCUSSION THREAD

**Discussion by khawthorne** [jazznbass/scan#142] [General]
What does the "uncorrected tau" output represent in the output from the corrected_tau() function? I haven't been able to figure this out from the documentation or from your very helpful ebook. The values don't seem to quite match any of the other Tau-Us (A vs. B, etc.). 

Thank you!

**Comment by jazznbass** [comment]
The uncorrected tau is the kendall's tau correlation between the values and phase.
```
> cor(as.numeric(exampleAB$Johanna[[1]]$phase), exampleAB$Johanna[[1]]$values, method = "kendall")
[1] 0.6299408
> corrected_tau(exampleAB$Johanna)$tau_uncorrected
  Johanna 
0.6299408 
```
